pkg/api/validation/validation.go

```diff
@@ -649,3 +649,29 @@ func ValidateResourceName(str string) errs.ValidationErrorList {
 
 	return errs.ValidationErrorList{}
 }
+
+// ValidateLimitRange tests if required fields in the LimitRange are set.
+func ValidateLimitRange(limitRange *api.LimitRange) errs.ValidationErrorList {
+	allErrs := errs.ValidationErrorList{}
+	if len(limitRange.Name) == 0 {
+		allErrs = append(allErrs, errs.NewFieldRequired("name", limitRange.Name))
+	} else if !util.IsDNSSubdomain(limitRange.Name) {
+		allErrs = append(allErrs, errs.NewFieldInvalid("name", limitRange.Name, ""))
+	}
+	if len(limitRange.Namespace) == 0 {
+		allErrs = append(allErrs, errs.NewFieldRequired("namespace", limitRange.Namespace))
+	} else if !util.IsDNSSubdomain(limitRange.Namespace) {
+		allErrs = append(allErrs, errs.NewFieldInvalid("namespace", limitRange.Namespace, ""))
```

Comment:
I replied in #3057 that I think it stays namespaced.

I think a ```ClusterLimitRange``` could be non-namespaced, and the admission control plug-in to enforce that is super simple, since it only has to pull 1 of these.  A ```CluserLimitRange``` would take precedence over what is presented in this PR and the actors that can define it are obviously different.

Prefer this PR stay limited to I want to enforce limits at the namesapce scope.